DependencyInjection/Configuration.php

```diff
@@ -316,6 +317,7 @@ public function getConfigTreeBuilder()
                     ->prototype('array')
                         ->fixXmlConfig('member')
                         ->fixXmlConfig('excluded_404')
+                        ->fixXmlConfig('excluded_http_code')
```

Comment:
excluded_http_code**s**

Reply:
No, this is exactly what we want. The singular named `excluded-http-code` XML element node will be transformed into a list names `excluded_http_codes`.